CHANGELOG_PENDING.md

```diff
@@ -7,6 +7,7 @@ Special thanks to external contributors on this release:
 ### BREAKING CHANGES:
 
 * CLI/RPC/Config
+- [httpclient] Update Subscribe interface to reflect new pubsub/eventBus API [ADR-33](https://github.com/tendermint/tendermint/blob/develop/docs/architecture/adr-033-pubsub.md)
```

Comment:
I think the Subscribe interface change is breaking on the Go API, but the new behaviour of non-blocking send is breaking on the RPC since now clients need to be fast or they'll get cut!